smf/ntp/manifest/manifest.xml

```diff
@@ -14,7 +14,7 @@
              type="service"
              version="1">
 
-        <create_default_instance enabled="false" />
+        <create_default_instance enabled='true' />
```

Comment:
If you're changing to single quotes here, perhaps make it consistent throughout?